pkgs/tools/networking/swagger-codegen/default.nix

```diff
@@ -11,8 +11,8 @@ stdenv.mkDerivation rec {
   ];
 
   src = fetchurl {
-    url = "https://repo1.maven.org/maven2/io/swagger/${pname}-cli/${version}/${jarfilename}";
-    sha256 = "06xx42ayh4xqpr71lq1hj7kv1v6m9ld9jm1d15fhs935zqckv32a";
+    url = "https://repo1.maven.org/maven2/io/swagger/codegen/v3/${pname}-cli/${version}/${jarfilename}";
+    sha256 = "0akkc0mdxp2mhn8dl3lwv6y11b718aza78cndn9p7736ffw5q6s0";
   };
 
   phases = [ "installPhase" ];
```

Comment:
Please do not overwrite phases and instead the one that should no run.

Reply:
I'm sorry, could you please elaborate on this?

Reply:
```suggestion
```

and add ``dontBuild = false;`` etc.

Reply:
Ah, sorry it took me so long, should be resolved now. Thanks!

Reply:
You messed something up while pushing this change.

Reply:
I've also changed the whole structure and move the package into new directory (swagger-codegen3) since I realized we might need both the current `swagger-codegen` (for v2) and this one. I've also changed the output binary name (appended `3` at the end) so that it would be possible to have both installed concurrently.

Reply:
What do you mean by messing it up? Is it related to ^^^ ?

Reply:
You did not change this line. phases should not be overwritten in leaf packages.

Reply:
Ah, gosh, I feel stupid now. I didn't notice that I've left v2 here as well (it changed in the meantime but I didn't do a rebase). Sorry!!!

Reply:
Should I create a new PR for the original `swagger-codegen` (to get rid of the `installPhase`) or would you like me to do it here?

Reply:
You can do that in this PR.

Reply:
Ok, hopefully fixed. Thanks a million for your patience.